File: linearSearch.c
#include <stdio.h>
int linearSearch(int arr[],int size,int element){
    for(int i = 0; i < size;i++){
        if(arr[i] == element){
            return i;
        }
    }
    return -1;
}

int main(){
    int arr[] = {1,2,3,22,33,44,11,587};
    int size,element,searchIndex;
    size = sizeof(arr) / sizeof(int);
    printf("Enter any element that you want to find:\n");
    scanf("%d",&element);
    searchIndex = linearSearch(arr,size,element);
    if(searchIndex != -1){
        printf("%d is at index %d\n",element,searchIndex);
    }
    else {
        printf("%d is not in the array\n",element);
    }
    return 0;
}

File: linearSearch.py
array = []
for _ in range(int(input("How many elements you want in your array\n"))):
    array.append(int(input()))
element = int(input("Enter an element that you want to find in your array\n"))
def linearSearch(arr,ele):
    for i in arr:
        if(i == ele):
            return i 
    return -1

# get the index of a given element
searchIndex = linearSearch(array,element)

if(searchIndex):
    print(f"Your {element} is at {searchIndex}\n")
else:
    print(f"{element} is not present in your array\n")
